tmail-backend/mailbox/team-mailboxes/pom.xml

```diff
@@ -14,6 +14,10 @@
     <name>Team-mail :: Mailbox :: Team-Mailboxes</name>
 
     <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>jmap-extensions</artifactId>
+        </dependency>
```

Comment:
Well, this sucks and makes circular dependent.

Will need to resolve this.

Reply:
Yes you can do that by defining a third party API